cl/opinion_page/urls.py

```diff
@@ -27,6 +28,22 @@
         view_docket,
         name="view_docket"
     ),
+    url(
+        r'''(?x)
+            ^recap/gov.uscourts\.
+            (?P<court>[^\./]+)\.
+            (?P<pacer_case_id>[^\./]+)/$''',
+        view_docket_recap,
+        name="view_docket_recap",
+    ),
+    url(
+        r'''(?x)
```

Comment:
The no slash thing will be handled automatically by Django. https://docs.djangoproject.com/en/2.0/ref/settings/#append-slash

No need for it here.

Reply:
Great. It's gone then.